modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2GetStarted.testcase

```diff
@@ -23,125 +23,196 @@ definition {
 	@priority = "5"
 	test IsStartProjectShowingInFirstTime {
-		CustomerPortal.showCPObject(title = "Account Flag");
+		task ("Creating objects") {
+			CustomerPortal.showCPObject(title = "Account Flag");
+		}
+
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
+
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		CustomerPortal.changeUserRole(
-			accountName = "Test Account 01",
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.viewGetStartedModal();
+		task ("Assert the Get Start modal") {
+			CustomerPortal.viewGetStartedModal();
+		}
 
-		CustomerPortal.deleteEntryAccountFlag(entry = "ERC-001");
+		task ("Delete the entries and objects created") {
+			CustomerPortal.deleteEntryAccountFlag(entry = "ERC-001");
 
-		CustomerPortal.deleteCPObject(title = "Account Flag");
+			CustomerPortal.deleteCPObject(title = "Account Flag");
+		}
 	}
 
-	@description = "LPS-145259 - Verify that the Get Started button takes the user to the Invite Your Team Members page."
+	@description = "Automation ID: LPS-145259 | Test Summary: Verify that the Get Started button takes the user to the Invite Your Team Members page"
 	@priority = "5"
 	test StartPojectButtonRedirectsToYourTeamMembers {
-		CustomerPortal.showCPObject(title = "Account Flag");
+		task ("Creating objects") {
+			CustomerPortal.showCPObject(title = "Account Flag");
+		}
 
-		CustomerPortal.changeUserRole(
-			accountName = "Test Account 01",
-			accountRoleTitle = "Account Administrator",
-			accountUserName = "Test Test");
+		task ("Change the user role in the project") {
+			Accounts.openAccountsAdmin();
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+			CustomerPortal.accountTableProject(title = "Test Account LXC-SM");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Administrator",
+				accountUserName = "Test Test");
+		}
 
-		CustomerPortal.clickOnStartProjectSetup();
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.viewInviteTeamMembersModalPage();
+			CustomerPortal.enterOnProject(title = "Test Account LXC-SM");
+		}
 
-		CustomerPortal.deleteEntryAccountFlag(entry = "ERC-001");
+		task ("Going through onboardig") {
```

Comment:
Luana, please check the typo: "onboarding"

Reply:
Done.